options/locale/locale_en-US.ini

```diff
@@ -1772,7 +1772,7 @@ pulls.auto_merge_has_pending_schedule = %[1]s scheduled this pull request to aut
 
 pulls.auto_merge_cancel_schedule = Cancel auto merge
 pulls.auto_merge_not_scheduled = This pull request is not scheduled to auto merge.
-pulls.auto_merge_canceled_schedule = The auto merge was canceled for this pull request.
+pulls.auto_merge_canceled_schedule = The auto merge was cancelled for this pull request.
```

Comment:
Actually, either is fine:

https://www.grammarly.com/blog/canceled-vs-cancelled/

> Canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one l), while cancelled (two l's) is preferred in British English and other dialects. Canceling/cancelling and canceler/canceller follow the same pattern.

Reply:
This is `en-US`, I'll replace `cancelled` in other places as well.